app/src/main/java/org/simple/clinic/contactpatient/ContactPatientUpdate.kt

```diff
@@ -46,6 +46,17 @@ class ContactPatientUpdate(
     }
   }
 
+  private fun overdueAppointmentLoaded(model: ContactPatientModel, event: OverdueAppointmentLoaded): Next<ContactPatientModel, ContactPatientEffect> {
+    val updatedModel = if (!model.hasLoadedPatientProfile) {
+      model.overdueAppointmentLoaded(event.overdueAppointment)
+    } else {
+      model.overdueAppointmentLoaded(event.overdueAppointment)
+          .contactPatientInfoLoaded()
```

Comment:
The commit message does not match with the update that we're doing in this commit. We're updating the model here and not the UI. Can you change the commit message to reflect that?

Reply:
That's intentional, I was keeping it consistent with the existing testing naming. Also when we update the model we update the UI anyway. For example, when the progress state gets changed we trigger the UI updates to show/hide the progress indicator here.

Reply:
I know it does happen but that's not happening in that particular commit. I understand that only when I review the commits after that. So it wasn't very clear to me.